src/FluentMigrator.Runner.Core/Generators/Generic/GenericQuoter.cs

```diff
@@ -30,9 +30,9 @@ public class GenericQuoter : IQuoter
     {
         private readonly QuoterOptions _options;
 
-        public GenericQuoter([CanBeNull] IOptions<QuoterOptions> options = null)
+        public GenericQuoter(IOptions<QuoterOptions> options)
```

Comment:
I see you did my comments in the previous commit.  Agree, using [CanBeNull] and assigning to null adds messy three-valued logic that we don't need.  **We just need a default constructor for GenericQuoter so that we don't break ABI with existing consumers.**

Reply:
`[CanBeNull]` for just for me in order that I don't have billions of compilation errors when I add this new parameter. But you're right, I should've added a default obsolete constructor.